tests/rules/property.js

```diff
@@ -145,7 +147,21 @@ eslintTester.run("property", rule, {
             ]
         },
 
-
+        // Typescript support valid cases
+        {
+            code: "lol.innerHTML = (5 as string);",
```

Comment:
Shouldnt that be allowed? AFAIU we allow literals in assignments to innerHTML.
Should be the same as `lol.innerHTML = 5` and that's definitively safe.

Reply:
I updated `TSAsExpressions` to the `checkAllowed` method [here](https://github.com/mozilla/eslint-plugin-no-unsanitized/pull/137/files#diff-69b92fc433cd9abc7aff2c0c2fd0e97aR77).

Added this to valid test cases.